react-components/hello-world.md

```diff
@@ -131,57 +132,45 @@ An example list of visible UI elements include:
 Our project should have the following structure:
 
 ```
+├── node_modules
 ├── public
-│   └── index.html
 ├── src
+│   ├── assets
 │   ├── App.css
-│   ├── App.js
-│   ├── App.test.js
+│   ├── App.jsx
 │   ├── index.css
-│   ├── index.js
-│   ├── logo.svg
-│   ├── reportWebVitals.js
-│   └── setupTests.js
+│   └── main.jsx
+├── index.html
```

Comment:
Include a line for .eslint.cjs, which is the lint specification that vite adds.

Unsurprisingly (given they don't use semicolons in their template), it doesn't have a warning for semicolons turned on. Maybe we can include the rule the reader could add to turn on that behavior in the semicolon callout below?